app/src/main/java/eu/kanade/presentation/track/components/TrackLogoIcon.kt

```diff
@@ -22,11 +22,13 @@ import tachiyomi.presentation.core.util.clickableNoIndication
 fun TrackLogoIcon(
     tracker: Tracker,
     onClick: (() -> Unit)? = null,
+    onLongClick: (() -> Unit)? = null,
 ) {
-    val modifier = if (onClick != null) {
-        Modifier.clickableNoIndication(onClick = onClick)
-    } else {
-        Modifier
+    val modifier = when {
+        onLongClick != null && onClick != null ->
+            Modifier.clickableNoIndication(onLongClick = onLongClick, onClick = onClick)
+        onClick != null -> Modifier.clickableNoIndication(onClick = onClick)
```

Comment:
You can pass in a null `onLongClick` value. So revert back to the previous if else and pass in `onLongClick` as is.